### PR TITLE
fixed labels in object manipulator scene

### PR DESCRIPTION
**Pull request by thalbern**
They were still referring to primitive manipulator instead of object manipulator

image

### CODE REVIEW

File: Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scenes/ObjectManipulatorExample.unity [hunk 1]
```diff
@@ -1301,69 +908,10 @@ Transform:
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
   m_LocalPosition: {x: 0, y: 0, z: 0}
   m_LocalScale: {x: 1, y: 1, z: 1}
-  m_Children:
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scenes/ObjectManipulatorExample.unity]: Feels like a lot of removed `GameObjects` in here, and this children list dropping to nothing.
Is this just a by-product of the updated serialization when you changed the text? Or were there actually removed GOs?

**Comment by cre8ivepark** [comment]
Made a small layout update - moved the text description onto the back panel, updated the text 'our example scene' to 'an example scene'.
Thanks!

File: Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scenes/ObjectManipulatorExample.unity (hunk 1 above)
Comment (thalbern) [reply Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/ObjectManipulator/Scenes/ObjectManipulatorExample.unity]: that was just a byproduct of saving the scene - i only changed the label text and didn't touch anything else - it's also removing the boundary system / spatial awareness etc.

**Pull request merged**